package.json

```diff
@@ -70,7 +70,8 @@
     "sinon": "^7.1.1"
   },
   "dependencies": {
-    "@folio/stripes-erm-components": "^5.0.0",
+    "@folio/stripes-erm-components": "^8.0.0",
+    "@k-int/stripes-kint-components": "^2.5.0",
```

Comment:
`"@k-int/stripes-kint-components": "^4.0.0"`

Reply:
Thanks! I changed k-int version to 4.0.0.